_overviews/scala3-migration/options-new.md

```diff
@@ -40,6 +40,21 @@ The current page only contains the options that were added in Scala 3.0.x.
 | `-Vprofile-sorted-by:<column-name>` | Show metrics about sources and internal representations sorted by given column name. |
 | `-Vprofile-details N` | Like -Vprofile, but also show metrics about sources and internal representations of the N most complex methods |
 
+## Warning settings
+
+| 3.3.x | description |
+|-|-|
+| `-Wunused:imports` | Warn about unused imports |
```

Comment:
weren't all -Wunused present in Scala 2? The same for value-discard.`Wnonunit-statement` seems to be new, same for `Wimplausible-patterns` and some others.

[ScalaSettings.scala#L169-L170](https://github.dev/lampepfl/dotty/blob/e96cb1866a03106662f2c7baef34e115bf8c9e6d/compiler/src/dotty/tools/dotc/config/ScalaSettings.scala#L169-L170)
https://docs.scala-lang.org/overviews/compiler-options/

Reply:
But -Wnounit-statement for scala2 is not documented in https://docs.scala-lang.org/overviews/compiler-options/

Reply:
That page is in a grievous state of unmaintainedness, I'm ashamed to say https://github.com/scala/docs.scala-lang/issues/1711